ballerina/Ballerina.toml

```diff
@@ -1,15 +1,15 @@
 [package]
 org = "ballerina"
 name = "xslt"
-version = "@toml.version@"
+version = "2.0.0"
 distribution = "slbeta3"
 
 [[platform.java11.dependency]]
-path = "../native/build/libs/xslt-native-@project.version@.jar"
-version = "@project.version@"
+path = "../native/build/libs/xslt-native-2.0.0-SNAPSHOT.jar"
+version = "2.0.0-SNAPSHOT"
```

Comment:
We don't need the version here, since we directly provide the jar path.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-xslt/pull/265/commits/2ceaf4f54619639193e9bcfce4cbe37e1532dfbd